### ISSUE TITLE
grey screen with ncurses-6.1-r2 on Gentoo

### ISSUE THREAD

**Issue by jktjkt** [a-n-t-h-o-n-y/CPPurses#9]
After applying the following patch to fix random segfaults due to mixing `ncursesw` with `tinfo` (instead of `tinfow`):
```diff
diff --git a/cmake/Modules/FindNcursesw.cmake b/cmake/Modules/FindNcursesw.cmake
index e572c70..4b098f7 100644
--- a/cmake/Modules/FindNcursesw.cmake
+++ b/cmake/Modules/FindNcursesw.cmake
@@ -127,9 +127,9 @@ if(CURSES_USE_NCURSESW)
   CHECK_LIBRARY_EXISTS("${CURSES_NCURSESW_LIBRARY}"
     cbreak "" CURSES_NCURSESW_HAS_CBREAK)
   if(NOT CURSES_NCURSESW_HAS_CBREAK)
-    find_library(CURSES_EXTRA_LIBRARY tinfo HINTS "${_cursesLibDir}"
-      DOC "Path to libtinfo.so or .lib or .a")
-    find_library(CURSES_EXTRA_LIBRARY tinfo )
+    find_library(CURSES_EXTRA_LIBRARY tinfow HINTS "${_cursesLibDir}"
+      DOC "Path to libtinfow.so or .lib or .a")
+    find_library(CURSES_EXTRA_LIBRARY tinfow )
   endif()
 
   # Report whether each possible header name exists in the include directory.
@@ -163,9 +163,9 @@ if(CURSES_USE_NCURSESW)
   endif()
 
 
-  find_library(CURSES_FORM_LIBRARY form HINTS "${_cursesLibDir}"
-    DOC "Path to libform.so or .lib or .a")
-  find_library(CURSES_FORM_LIBRARY form )
+  find_library(CURSES_FORM_LIBRARY formw HINTS "${_cursesLibDir}"
+    DOC "Path to libformw.so or .lib or .a")
+  find_library(CURSES_FORM_LIBRARY formw )
 
   # Need to provide the *_LIBRARIES
   set(CURSES_LIBRARIES ${CURSES_LIBRARY})
```
I tried to run the demo. All I got was a blank, grey screen. There was no text, no menus, no widgets. Tried in both Konsole (which sets `TERM=xterm-256color`) and plain old XTerm (`TERM=xterm`).

**Comment by a-n-t-h-o-n-y** [comment]
I was able to duplicate this on a Lubuntu VM, and I have a fix on the master branch for this(9aa6895f14fd0c0945836a4b8fb2334f689d4f85). In the case of Lubuntu the non-wide ncurses header was included and the wide curses library linked in. Let me know if this fixes your issue.
As for tinfow, some distros include tinfo in the ncurses lib and do not have a tinfow library easily available so I'm not confident on a solution for this that doesn't break other builds.

**Comment by jktjkt** [comment]
Thanks, recent master now builds without extra patches and something is visible on screen.
